packages/core/src/payment/strategies/bolt/bolt-payment-strategy.ts

```diff
@@ -84,6 +84,8 @@ export default class BoltPaymentStrategy implements PaymentStrategy {
     }
 
     deinitialize(): Promise<InternalCheckoutSelectors> {
+        this._embeddedField?.unmount();
```

Comment:
Do we need to update tests to reflect this behaviour?

Reply:
Added tests for this behaviour